docs/configmaps.md

```diff
@@ -0,0 +1,15 @@
+# ConfigMaps
+HCO creates ConfigMaps on deployment for supplying default values configuration.
+
+## Storage ConfigMap
+Defines default values for storage specs: 'accessMode' and 'volumeMode'.
+
+### Example
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: kubevirt-storage-class-defaults
+  namespace: openshift
+data:
+  accessMode: ReadWriteMany
+  volumeMode: Filesystem (Block for BareMetal infrastructure)
```

Comment:
nit: the example should be copy&paste, so I suggest to add the hint into a valid comment